features/definitions/resolvers-steps.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const should = require('should'),
+  { When, Then } = require('cucumber');
+
+const baseRequest = {
+  controller: 'kuzzle-plugin-graphql/graphql',
+  action: 'endpoint',
+  refresh: 'wait_for',
```

Comment:
I don't think you need to specify this since your route does not seems to use it

Reply:
You mean the **'refresh'** property?